plugins/nodes/src/js/pages/nodes/nodes-grid/NodesGridContainer.js

```diff
@@ -21,9 +22,8 @@ class NodesGridContainer extends mixin(StoreMixin, QueryParamsMixin) {
     this.state = {
       filteredNodes: [],
       filters: {health: 'all', name: '', service: null},
-      hasLoadingError: false,
+      mesosXHRError: null,
       hiddenServices: [],
-      mesosStateErrorCount: 0,
```

Comment:
The error could have a status code other than 500. We shouldn't be trying again if it's a 401 or 403 for example. This error handling (API errors kept in component state) is not a good pattern and makes handling errors contextually much more complex. This retry feature should be handled closer to the API.